lib/defaults.js

```diff
@@ -48,7 +48,7 @@ var DefaultDescriptor = {
     prompt: false
   },
   netrc: {
-    name: 'netrc',
+    name: 'Netrc',
```

Comment:
Minor nit, this change isn't related to what you're trying to accomplish so I'd rather it not be in this PR.

Reply:
Done